custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -0,0 +1,211 @@
+{
+    "meta": {
+       
+        "label": "Ontology tagging",
+        "description": "Tag documents matching keywords within a corpus of text documents",
+        "icon": "icon-pushpin"
+    },
+
+    "kind": "PYTHON",
+   
+
+    "inputRoles": [
+        {
+            "name": "Document dataset",
+            "label": "Document dataset",
+            "description": "Dataset with a column containing the text documents to tag",
+            "arity": "UNARY",
+            "required": true,
+            "acceptsDataset": true
+        },
+        {
+            "name": "Ontology dataset",
+            "label": "Ontology dataset",
+            "description": "Dataset with columns for tags, keywords and categories",
+            "arity": "UNARY",
+            "required": true,
+            "acceptsDataset": true
+        }
+    
+    ],
+
+    "outputRoles": [
+        {
+            "name": "Tagged documents",
+            "label": "Tagged documents",
+            "description": "Dataset with the input documents and their assigned tags",
+            "arity": "UNARY",
+            "required": true,
+            "acceptsDataset": true
+        }
+  
+    ],
+    
+    "selectableFromDataset": "Document dataset",
+    "selectableFromDataset":"Ontology dataset",
```

Comment:
We can only choose one selectable dataset - in the current case, the last one prevails.
```suggestion
```